### DISCUSSION TITLE
Expose Tokens defined in a template

### DISCUSSION THREAD

**Discussion by ujjal-Neudesic** [Handlebars-Net/Handlebars.Net#431] [Q&A]
Any way to find our Tokens inside a template?  am building a preview functionality and need tokens to be displayed.

**Comment by oformaniuk** [comment]
Hello @ujjal-Neudesic 
Unfortunately there's no way to do it at the moment. You can track the following feature request https://github.com/Handlebars-Net/Handlebars.Net/issues/369
